src/apidefs/hello.world.api.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: loopback-next-hello-world
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+"use strict";
```

Comment:
We use single quotes in our code, please fix all other places in this file too.

I think you may need to configure your VSCode to use single quotes instead of Microsoft's default double quotes?